Update the advanced settings editor tab (#4730)
qgis/QGIS-Documentation: source/docs/user_manual/introduction/qgis_configuration.rst

Advanced Settings
-----------------

.. _figure_advanced_settings:

.. figure:: img/options_advanced.png
   :align: center

   Advanced Settings tab in QGIS

All the settings related to QGIS (UI, tools, data providers, Processing
configurations, default values and paths, plugins options, expressions,
geometry checks...) are saved in a :file:`QGIS/QGIS3.ini` file under the active
:ref:`user profile <user_profiles>` directory.

> Pasting this file among QGIS installations also share the configurations.

Reviewer: ```suggestion
Configurations can be shared by copying this file to other installations.
```

From within QGIS, the :guilabel:`Advanced` tab offers a way to manage these
settings through the :guilabel:`Advanced Settings Editor`.
After you promise to be careful, the widget is populated with a tree of all

> the existing settings, whose value you can directly edit.

Reviewer: ```suggestion
the settings, and these can all be modified.
```

Author: hmmm... Issue here is that the "tree" has four columns and only the value column can be modified; you can not change setting name, type or fill a description, so claiming that *these can all be modified*, *these* not indexing anything particular, is a bit too much imho.

Reviewer: I gave it another go.  Not sure if it is better, though...

> Right-click over a setting or a group and you can delete it.

Reviewer: ```suggestion
Right-click over a setting or a group and you can delete it
(to add a setting or group, you have to edit the QGIS3.ini file).
```
